episode-6.msrt

```diff
@@ -254,7 +252,7 @@
 51
 00:03:43,310 --> 00:03:47,310
 [ru] Зелье Зомбификации... глупости какие!
-[en] A potion of Zombification... what nonsense!
+[en] A potion of Zombification... that's nothing!
```

Comment:
Much thanks to everyone for explanations! I agree with @craigmaloney's arguments, so lets keep "that's nothing". ^__^